specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -5189,6 +5203,48 @@
       },
       "description": "Response from a List SynonymMaps request. If successful, it includes the full definitions of all synonym maps."
     },
+    "EncryptionKey": {
+      "properties": {
+        "keyVaultKeyName": {
+          "type": "string",
+          "description": "The name of the KeyVault key."
+        },
+        "keyVaultKeyVersion": {
+          "type": "string",
+          "description": "The version of the KeyVault key."
+        },
+        "keyVaultUri": {
+          "type": "string",
+          "description": "The URI of the KeyVault."
```

Comment:
This becomes public documentation. The bar is very, very high, so please include enough context to make these useful.

Reply:
I re-used the description from the official Storage KeyVaultProperties sdk -> https://github.com/Azure/azure-sdk-for-net/blob/master/src/SDKs/Storage/Management.Storage/Generated/Models/KeyVaultProperties.cs

I agree we can (and should) be more detailed, but I found it hard to write elaborate description of simple properties such as name/version without ending up adding a lot of superfluous words